test/fixtures/bn.json

```diff
@@ -0,0 +1,105 @@
+{
+  "valid": [
+    {
+      "dec": "0",
+      "hex": ""
```

Comment:
I'd like to choose the path of least surprise. bn.js is used not only for crypto purposes, but for UI input/output too.